src/main/java/com/appdynamics/extensions/azure/customnamespace/AzureCustomNamespaceMonitorTask.java

```diff
@@ -159,7 +163,7 @@ private void collectStatistics() {
             List<FutureTask<List<Metric>>> targetFutureTask = buildFutureTargetTasks(executorService, account.getTargets());
             metrics = CommonUtilities.collectFutureMetrics(targetFutureTask, config.getConcurrencyConfig().getThreadTimeout(), "AzureCustomNamespaceMonitorTask");
         } catch (Exception e) {
-            LOGGER.error("Exception occurred while collecting target metrics", e);
+            LOGGER.error("[ACCOUNT={}] - Exception occurred while collecting metrics for targets ||{}||", account.getDisplayName(), targets, e);
```

Comment:
nit: 
adding `targets` in the logging statement will log the object(_objectId for the List<Target>_) and probably will not be helpful.

Reply:
I see.  You are correct.  I fix and push an update.